controller/common/src/Controller/Common/Order/Standard.php

```diff
@@ -430,45 +430,53 @@ protected function updateStockBundle( string $prodId, string $stockType )
 	/**
 	 * Updates the stock levels of selection products for a specific type
 	 *
-	 * @param string $prodId Unique product ID
-	 * @param string $stocktype Unique stock type
+	 * @param \Aimeos\MShop\Order\Item\Base\Product\Standard $item
 	 * @return \Aimeos\Controller\Common\Order\Iface Order controller for fluent interface
 	 */
-	protected function updateStockSelection( string $prodId, string $stocktype )
+	protected function updateStockSelection( $item )
```

Comment:
Can you add the parameter type `\Aimeos\MShop\Order\Item\Base\Product\Iface` too?